### PR TITLE
Improve fix to persona coin initials bug to align with fix for 5.x

### PR DESCRIPTION
**Pull request by qianghuang94**
#### Pull request checklist

- [x] Addresses an existing issue: Fixes https://github.com/OfficeDev/office-ui-fabric-react/issues/6570
- [x] Include a change request file using `$ npm run change`

#### Description of changes
Original the issue was fixed by https://github.com/OfficeDev/office-ui-fabric-react/pull/6584 for version 6.x 
and i created a PR to fix the issue in 5.x
https://github.com/OfficeDev/office-ui-fabric-react/pull/7027

The fix contains in https://github.com/OfficeDev/office-ui-fabric-react/pull/6584 is not good enough in some cases, as a result of that i created this PR to improve it.

### CODE REVIEW

**Comment by qianghuang94** [comment]
@dzearing @natalieethell Could you please also help to review this pr?

File: packages/office-ui-fabric-react/src/components/Persona/PersonaCoin/PersonaCoin.base.tsx [hunk 1]
```diff
@@ -95,30 +96,30 @@ export class PersonaCoinBase extends BaseComponent<IPersonaCoinProps, IPersonaSt
...
-            <PersonaPresence {...personaPresenceProps} />
-          </div>
-        ) : // Otherwise, render just PersonaPresence.
-        this.props.presence ? (
-          <PersonaPresence {...personaPresenceProps} />
-        ) : (
-          // Just render Contact Icon if there isn't a Presence prop.
-          <Icon iconName="Contact" className={classNames.size10WithoutPresenceIcon} />
-        )}
+          size !== PersonaSize.size10 && size !== PersonaSize.tiny ? (
```
Comment (mtennoe) [inline packages/office-ui-fabric-react/src/components/Persona/PersonaCoin/PersonaCoin.base.tsx]: What happened with the whitespace here?

**Comment by mtennoe** [comment]
Can we get some tests for this scenario?

**Comment by mtennoe** [comment]
So this fix is already in 5.x?

File: packages/office-ui-fabric-react/src/components/Persona/PersonaCoin/PersonaCoin.base.tsx (hunk 1 above)
Comment (qianghuang94) [reply packages/office-ui-fabric-react/src/components/Persona/PersonaCoin/PersonaCoin.base.tsx]: Removed the whitespaces, thanks

**Comment by qianghuang94** [comment]
@mtennoe there is test 

> So this fix is already in 5.x?

Nope, but i created a PR to fix this issue in 5.x https://github.com/OfficeDev/office-ui-fabric-react/pull/7027

**Comment by qianghuang94** [comment]
> Can we get some tests for this scenario?

We already have test to test one scenario, the new condition checking is an enhancement to original fix https://github.com/OfficeDev/office-ui-fabric-react/pull/6584

**Comment by qianghuang94** [comment]
One screener test failed, can anyone help to find corresponding code so that i could find the root cause?

**Comment by natalieethell** [comment]
I'm not sure what's going on with the screener test here. I'll try closing and opening this PR again.

**Comment by natalieethell** [comment]
Can you try merging with master again?

**Comment by natalieethell** [comment]
Nevermind, looks like it's been resolved.

**Comment by dzearing** [comment]
ah ha here's the remaining fix.

**Pull request merged**